vue/sbc-common-components/src/util/constants.ts

```diff
@@ -32,7 +32,9 @@ export enum LoginSource {
 
 export enum Role {
   AccountHolder = 'account_holder',
-  PublicUser = 'public_user'
+  PublicUser = 'public_user',
+  STAFF = 'staff',
```

Comment:
Can you rename STAFF as Staff to keep consistent with other enum keys in Role ?

Reply:
updated